sandbox/network.tf

```diff
@@ -65,6 +65,8 @@ resource "aws_security_group" "docker_daemon" {
       "10.1.10.0/24",
       "10.1.20.0/24",
       "10.1.30.0/24",
+      "100.64.0.0/10",
+      "172.20.0.0/16",
```

Comment:
`100.64.0.0/10` and `172.20.0.0/16` can be retrieved from https://github.com/gliderlabs/infra.gl/blob/master/manifold/cluster/config#L25 and https://github.com/gliderlabs/infra.gl/blob/master/manifold/cluster/config#L28